assets/cms/components/IconButton.vue

```diff
@@ -0,0 +1,70 @@
+<template functional>
+    <span class="button-container">
+        <button
+            @click='listeners.click'
+            v-bind:class="['btn', props.bootstrapClass]"
+            :disabled="props.disabled"
+            >
+            <span class="label" v-if="props.labelPosition === 'left'"
+                v-bind:class="{disabled : props.disabled}">
+                <slot />
+            </span>
+
+            <Icon :icon="props.icon" :type="props.type" :color="props.disabled ? '#ccc' : props.color"/>
+
+            <span class="label" v-if="props.labelPosition === 'right' || !props.labelPosition"
+                v-bind:class="{disabled : props.disabled}">
+                <slot />
+            </span>
+
+        </button>
+    </span>
+</template>
+
+<script>
+import Icon from './Icon'
+
+export default {
+    props: {
+        outline: Boolean,
+        disabled: Boolean,
```

Comment:
This should have default: true

Reply:
I defaulted disabled to false because we dont want a button to be disable out of the box, and also removed the outline prop.